flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1136,6 +1136,7 @@
     },
     "io.github.realmazharhussain.GdmSettings": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
+	"finish-args-unnecessary-xdg-config-access": "required for 'apply current display settings to GDM' feature",
```

Comment:
Can you please fix the indentation here?

![image](https://github.com/flathub-infra/flatpak-builder-lint/assets/62639087/dd1bbc1a-1399-4f5a-bef1-7ead014b67ed)

Reply:
Sorry! Fixed it now.